src/main/kotlin/uk/gov/dluhc/printapi/client/BankHolidayDataFileClient.kt

```diff
@@ -0,0 +1,49 @@
+package uk.gov.dluhc.printapi.client
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.databind.type.MapType
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.core.io.Resource
+import org.springframework.stereotype.Component
+import java.time.LocalDate
+import javax.annotation.PostConstruct
+
+@Component
+class BankHolidayDataFileClient(
```

Comment:
I've added a sub-task to [EIP1-3669](https://technologyprogramme.atlassian.net/browse/EIP1-3669) to improve this in future (so that we're retrieving up-to-date bank holiday data from one place).